backend/internal/model/game.go

```diff
@@ -183,7 +183,6 @@ func (g *game) GetGameStats() GameStats {
 	log.Trace("mutex lock: GetGameStats")
 	g.m.RLock()
 	defer g.m.RUnlock()
-
 	return g.gameData
```

Comment:
```suggestion

	return g.gameData
```

Reply:
What commit caused game.go to be changed in this PR?

Reply:
5e90562a5c2f0a926f7e4cf0ca7e5f96571bf673 this one probably